### ISSUE TITLE
No document about CURLOPT_SAFE_UPLOAD

### ISSUE THREAD

**Issue by ve3** [php/doc-en#1622]
From this [`curl_setopt()`](https://www.php.net/manual/en/function.curl-setopt.php) page, there is no description about `CURLOPT_SAFE_UPLOAD` option.

It was introduced since [PHP 5.5 (maybe)](https://wiki.php.net/rfc/curl-file-upload) but did not found anywhere on php.net website (English version).

I found this [bug reported](https://bugs.php.net/bug.php?id=66677&edit=1) but still not see anywhere in English version.
Found in [Spanish](https://www.php.net/manual/es/function.curl-setopt.php), [Chinese](https://www.php.net/manual/zh/function.curl-setopt.php).

**Comment by cmb69** [comment]
Ugh, that's ugly! That option defaults to true as of PHP 5.6.0, and that default can't be changed as of PHP 7.0.0; as such, the option no longer makes sense, but apparently it has been kept for BC reasons (although it probably should have been removed altogether for PHP 8.0.0).

Anyhow, I think we need to bring back the documentation; something like:
````xml
          <row>
           <entry valign="top"><constant>CURLOPT_SAFE_UPLOAD</constant></entry>
           <entry valign="top">
            Always &true;, what disables support for the <literal>@</literal> prefix for
            uploading files in <constant>CURLOPT_POSTFIELDS</constant>, which
            means that values starting with <literal>@</literal> can be safely
            passed as fields. <classname>CURLFile</classname> may be used for
            uploads instead.
           </entry>
          </row>
````

**Comment by cmb69** [comment]
Fixed as 4c5d201d5f1a368c9ec6e81f53d16e8b2607d5c1.
